select/selectSearchable.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import SelectSearchable from "./selectSearchable";
+import { items } from "../app";

```

Comment:
this import is wrong, the items mock is in "../test/mocks/items"

Reply:
yes, i'm adjusting